src/main/java/no/sikt/nva/scrapers/SubjectScraper.java

```diff
@@ -37,6 +38,7 @@ public static List<String> extractTags(DublinCore dublinCore) {
                    .filter(SubjectScraper::isSubjectAndNotSpecificallyIgnored)
                    .map(DcValue::scrapeValueAndSetToScraped)
                    .filter(Objects::nonNull)
+                   .map(value -> value.replace("TermSet Emneord::", StringUtils.EMPTY_STRING))
```

Comment:
These might be from a controlled vocabulary. Before proceeding, I'd ask if this is a vocab they wish to maintain